src/foam/box/HTTPAuthorizationType.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.ENUM({
+  package: 'foam.box',
+  name: 'HTTPAuthorizationType',
+
+  documentation: 'Types of HTTP Athorization',
+
+  values: [
+    {
+      name: 'NONE',
+      label: 'None',
+      ordinal: 0
+    },
+    // {
```

Comment:
wondering why this is commented out?

Reply:
Can't decide if None and Basic are the same.